hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -381,6 +387,57 @@ private boolean bootstrapFromFilesystem(HoodieEngineContext engineContext, Hoodi
     return partitionToFileStatus;
   }
 
+  /**
+   * Initialize shards for a partition.
+   *
+   * Each shard is a single log file with the following format:
+   *    <fileIdPrefix>ABCD
+   * where ABCD are digits. This allows up to 9999 shards.
+   *
+   * Example:
+   *    fc9f18eb-6049-4f47-bc51-23884bef0001
+   *    fc9f18eb-6049-4f47-bc51-23884bef0002
```

Comment:
When an executor receives more than one file's worth of records, we end up creating f1-0_wt_cx.parquet, f1-1_wt2_cx.parquet etc.   Should the same naming convention used here?  

In other words, should you use the 36 byte fileId prefix + "-ABCD"?